### ISSUE TITLE
Import Error

### ISSUE THREAD

**Issue by EvgenyK1975** [streetwriters/notesnook-importer#110]
When I import files from evernote HTML or ENEX I got error message. Pls see samples 
![Error](https://github.com/streetwriters/notesnook-importer/assets/155667096/85eb24e0-46d2-4259-bfb6-5746a56cefc6)
[notesnook-importer (3).zip](https://github.com/streetwriters/notesnook-importer/files/13833770/notesnook-importer.3.zip)

**Comment by smikwily** [comment]
Getting this as well. I'm only processing a single file, so I'm only getting one error. I've tried HTML and MD and both are doing the same. Attached is HTML.

[notesnook-importer.zip](https://github.com/streetwriters/notesnook-importer/files/13835632/notesnook-importer.zip)

**Comment by adaspender** [comment]
Similar issue here. I'm on Linux Mint coming from Evernote and used the importer following all the [instructions](https://help.notesnook.com/importing-notes/import-notes-from-evernote), but the application sends out a TypeError hundreds of times:
![image](https://github.com/streetwriters/notesnook-importer/assets/118388731/5203f28b-4608-4a70-a5fd-983320fd56b7)
While the web app says st.name is undefined for the same .zip:
![image](https://github.com/streetwriters/notesnook-importer/assets/118388731/1eb3e727-e083-4a6e-9c45-ca5f0962defd)

I'm unfamiliar with the .enex format Evernote exports as, but looking at the files I do see them obfuscating all the written material into Base64 and quite a few titles have not exported out of Evernote, leaving only `<title>Untitled Note</title>` in the markup. Not sure if that helps better determine what's happening or not.

**Comment by adaspender** [comment]
@smikwily @EvgenyK1975 
Found a workaround, here's exactly what I did (on Mint):

1. Installed PipX to download and utilize [evernote-backup](https://github.com/vzhd1701/evernote-backup) to export my notes.

I had previously exported via Evernote's advertised Windows10 Desktop application, individually exporting my notebooks as per the instructions on [Notesnook's help page](https://help.notesnook.com/importing-notes/import-notes-from-evernote). When exporting through the Desktop application the files denote a version 10.71.2. When exporting through the evernote-backup script the files denote a version 10.10.5 and all notes were correctly titled in the markup.

2. Opened v2.2.1 of the [Notesnook Importer](https://importer.notesnook.com/) in an incognito Brave tab to ensure none of my add-ons were running and disabled Brave Shields (my vpn was still active).
3. Left Telemetry enabled, selected Evernote from the dropdown, uploaded my evernote-backup script's exported .enex files, then downloaded the importer generated .zip file.
4. Opened https://app.notesnook.com/ in an additional Brave incognito tab (again turning Shields off) and performed the import. This time I was told I needed to upgrade to import in the midst of doing so, but the import finished anyways.

That's it, praying for ya
